tutorials/tts/MixerTTS_FastPitch_Training.ipynb

```diff
@@ -148,10 +156,87 @@
   },
   {
    "cell_type": "markdown",
-   "id": "24fa15c0",
+   "id": "d12262e7",
+   "metadata": {},
+   "source": [
+    "### FastPitch\n",
+    "\n",
+    "FastPitch is  a fully-parallel text-to-speech model based on FastSpeech, conditioned on fundamental frequency contours. The model predicts pitch contours during inference. Conditioning on frequency contours improves the overall quality of synthesized speech. For more details about model, please refer to the original [paper](https://arxiv.org/abs/2006.06873).\n",
+    "\n",
+    "FastPitch like most NeMo models is defined as a LightningModule.\n",
+    "\n",
+    "Let's take a look using NeMo's pretrained model and how to use it to generate spectrograms."
```

Comment:
May also be worth pointing out where the procedure differs from Mixer-TTS so users can better learn how to generalize to different models in the future